### ISSUE TITLE
Make a CLI version of cobalt

### ISSUE THREAD

**Issue by RGBCube** [wukko/cobalt#254]
A simple CLI tool that is used like `cobalt <url>` would be great, used for downloading stuff to the cli. it would print the contents of the video/thing there at the url to stdout so it could integrate with the shell better.

plus it would be all local for speed and to not rely on external services.

**Comment by jackra1n** [comment]
why not use [yt-dlp](https://github.com/yt-dlp/yt-dlp)?

**Comment by RGBCube** [comment]
does that support Instagram / Twitter / etc etc?

**Comment by jackra1n** [comment]
> does that support Instagram / Twitter / etc etc?

@RGBCube yes. it supports hundreds if not thousands of websites: https://github.com/yt-dlp/yt-dlp/blob/master/supportedsites.md

**Comment by RGBCube** [comment]
> > does that support Instagram / Twitter / etc etc?
> 
> @RGBCube yes. it supports hundreds if not thousands of websites: https://github.com/yt-dlp/yt-dlp/blob/master/supportedsites.md

Oh, I did not know that. Then this issue is resolved. Thanks a lot

**Comment by dumbmoron** [comment]
could you reopen this issue please? while the above might solve your use case, this is a feature that I think is worthwhile to implement

**Comment by falcie1337** [comment]
duuude i was just thinking into trying to make something like that, so i scoured through the issues and found this, i totally agree with this, it would only make the project better overall in my honest opinion

**Comment by princessmortix** [comment]
I think I could do one.

**Comment by princessmortix** [comment]
Done, check it out: https://github.com/princessmortix/cobalt/releases/tag/v1.0.1

**Comment by princessmortix** [comment]
Thanks! (if you like the project, consider giving it one :star: ;)

**Comment by falcie1337** [comment]
> Thanks! (if you like the project, consider giving it one ⭐ ;)

i've already did so, i'm thinking into trying to mantain it as a AUR package too, i think it would be pretty cool to have it on there, never did it before so it would be a cool learning experience

**Comment by princessmortix** [comment]
Sure, I'd be awesome. I think I'll add to scoop once it reaches 50 stars (I think thats the minimum).

**Comment by authorisation** [comment]
Wasn't able to find an AUR package for it, so I made one

https://aur.archlinux.org/packages/cobalt-cli-git

**Comment by lostdusty** [comment]
Thank you!

**Comment by crckrberries** [comment]
could you also add it to [Homebrew](https://brew.sh)?
